style.css

```diff
@@ -338,3 +366,101 @@ html {
   color: #fff;
   transition: 0.3s;
 }
+
+@media screen and (min-width: 768px) {
+  .container {
+    margin: auto 10%;
+  }
+
+  /* NAVBAR SECTION  */
+  .navbar {
+    padding: 3.2rem 8.8rem;
+  }
+
+  .nav-btn {
+    display: none;
+  }
+
+  .nav-list {
+    list-style: none;
+    display: flex;
+    gap: 3.2rem;
+  }
+
+  .nav-list a:link,
+  .nav-list a:visited {
+    text-decoration: none;
+    color: #344563;
+    font-size: 1.5rem;
+    font-weight: 500;
+  }
+
+  .nav-list a:hover,
+  .nav-list a:active {
+    border-bottom: 0.3rem solid #6070ff;
+  }
+
+  /* HEADLINE SECTION  */
+  .headline {
+    background: url(images/Desktop/Headline-bg.svg);
+    background-repeat: no-repeat;
+    background-size: cover;
+    background-position: auto auto;
+    margin: 6.4rem 4.8rem;
+  }
+
+  /* PROJECT SECTION  */
+
+  .projects {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    column-gap: 4.8rem;
+    padding: 2.8rem;
+  }
+
+  .project-img {
+    align-self: center;
+  }
+
+  .projects:nth-child(even) > .project-img {
+    order: 2;
+  }
+
+  .btn {
+    justify-self: left;
+  }
+
+  /* ABOUT SECTION  */
+  .languages-elements {
+    flex-direction: row;
+  }
+
+  .languages-elements li {
+    flex-direction: column;
+    align-items: flex-start;
+    width: 12rem;
+    gap: 1.8rem;
+  }
+
+  /* CONTACT SECTION  */
+  .contact {
```

Comment:
- [ ] Kindly decrease the width of the paragraph in the contact section. 🙂️
- [ ] Please use sharp corners for the inputs in the desktop version. 🥸️
- [ ] I think it would be great if you increase the width of the input fields to match the design. 🧐️

**Your design**
![Screenshot from 2023-05-04 09-44-38](https://user-images.githubusercontent.com/17243325/236135321-649f5d1d-2a5a-4811-a3b4-b1a099ad36a5.png)

**Figma design**
![Screenshot from 2023-05-04 09-45-11](https://user-images.githubusercontent.com/17243325/236135397-2c38ebc3-2946-40d6-bccd-8eb9cb4deac8.png)